http/oidc/src/main/java/org/wildfly/security/http/oidc/JWTClientCredentialsProvider.java

```diff
@@ -115,7 +109,7 @@ public void init(OidcClientConfiguration oidcClientConfiguration, Object credent
             clientKeyAlias = oidcClientConfiguration.getResourceName();
         }
 
-        KeyPair keyPair = loadKeyPairFromKeyStore(clientKeyStoreFile, clientKeyStorePassword, clientKeyPassword, clientKeyAlias, clientKeyStoreType);
+        KeyPair keyPair = new JWTSigningUtils().loadKeyPairFromKeyStore(clientKeyStoreFile, clientKeyStorePassword, clientKeyPassword, clientKeyAlias, clientKeyStoreType);
```

Comment:
Once the methods from this utility class are updated to be static, you'll be able to use JWTSigningUtils.loadKeyPairFromKeyStore.